docs/source/markdown/podman-container-checkpoint.1.md

```diff
@@ -117,8 +117,7 @@ The default is **false**.
 
 #### **--latest**, **-l**
 
-Instead of providing the *container ID* or *name*, use the last created *container*. If other methods than Podman are used to run *containers* such as `CRI-O`, the last started *container* could be from either of those methods.\
-The default is **false**.\
+Instead of providing the *container ID* or *name*, use the last created *container*. The default is **false**.
```

Comment:
Why did you remove the CRI-O bit?  It needs to be rewritten, but I think it still holds.

"If tools other than Podman are used to run a *container*, such as `CRI-O` or another container engine, the last started *container* could be from one of those engines.

Reply:
cri-o does not create containers that --last would refer to.  These containers are in the podman database, CRI-O does not use the Podman database.